articles/working-with-forms-in-php/index.md

```diff
@@ -0,0 +1,106 @@
+### Introduction
+
+PHP forms have many usages. Ranging from contact forms, login forms, and registration forms for collecting and updating data in a web-based application. Forms are the fundamental interface between the user and the server.
+
+Creating a form on a web application is achieved using HTML. PHP connects the web application with the server (My SQL). It takes data from a webpage and sends it to the server. PHP handles data in the form. Data is then saved in the database, such as My SQL.
+
+Through this article, you will learn how to:
+
+- Create a PHP form using HTML
+- Understand [POST and GET methods](https://www.w3schools.com/tags/ref_httpmethods.asp). Learn how the two methods work and how they are different from each other.
+
+### Prerequisites
+
+Forms often take input from users. Therefore, the reader should have prior knowledge of HTML and PHP.
+
+### Overview of Forms
+
+A [form](https://www.tutorialspoint.com/php/php_form_introduction.htm) is a document with blank spaces for a user to insert or select data. The user's data is stored or saved in the database and retrieved anytime and anywhere needed.
+
+Collecting user's data in a Web application using forms is a major task. In web application development, there are many forms, some of them include:
+
+- Contact Form
+- Search Forms
+- Login Forms
+- Registration Forms.
+
+The Form is presented to the user. The user inserts data and submits the form using the submit button. Then the data is saved and processed by the server. Data is processed based on the method mentioned. There are two methods, GET and POST. Both methods send data to the server. The POST method is certain because the data sent is hidden.
+
+### How to Create PHP forms Using HTML
+
+Forms in PHP are similar to forms built using HTML besides the syntax used. The form contains `Html` tags with [Graphical User Interface (GUI)](https://www.computerhope.com/jargon/g/gui.htm) such as [buttons](https://www.w3schools.com/tags/att_button_form.asp) and [checkboxes](https://www.w3schools.com/tags/att_input_type_checkbox.asp). It therefore, becomes easy for the user to interact with the webpage or with the form. Forms are created for user-friendly purposes. Whereby a user with no technical skills will easily use it.
+
+Forms are written inside form tags `<form>` and `<\form>;.` The `form` tag surrounds all the inputs. Also, it gives instructions about how and where to submit the form. `POST` and `GET` methods declare where to submit the data.
```

Comment:
Instead of `Forms are written inside form tags `<form>` and `<\form>;``
Say
`Forms are defined by the <form></form> tags.`